docs/guidance/map/contract_suspension.md

```diff
@@ -0,0 +1,27 @@
+# Contract suspension
+
+A contract can be suspended after it is signed. A suspended contract is still legally in force, but work has been suspended. For example, a contract might be suspended in the following scenarios:
+
+* the buyer suspends a contract because the supplier fails to perform.
+* the supplier suspends a contract because the buyer fails to pay.
+* either party suspends a contract due to force majeure.
```

Comment:
Up to here, it sounds like the contract could just be 'terminatedEarly'. We'll need to explain a bit more about suspension.

Reply:
The first paragraph from https://www.upcounsel.com/contract-suspension is clear:

> A contract suspension is the temporary cessation of performance. It's not the same as [...] termination, which is a permanent cessation of performance.

Reply:
Good catch. I've clarified the description in 17c63f726790742d83b69b30c353730c8047a6b9